lib/keystone.lua

```diff
@@ -109,26 +108,25 @@ function Client:_updateToken(callback)
     end
 
     local function handleTokenResponse(res)
-      local data = ''
+      local chunks = {}
       res:on('data', function(chunk)
-        data = data .. chunk
+        table.insert(chunks, chunk)
```

Comment:
Just curious - is inserting into table faster in new version of luvit compared to concatenating a string? Or is this just a style / consistency change?